src/datadoc_editor/frontend/components/builders.py

```diff
@@ -183,49 +187,30 @@ def build_variables_machine_section(
 
 
 def build_variables_pseudonymization_section(
-    metadata_inputs: list,
-    title: str,
-    variable: model.Variable,
-    pseudonymization: model.Pseudonymization,
+    title: str, variable: model.Variable, selected_algorithm: str
```

Comment:
```suggestion
    title: str, variable: model.Variable, selected_algorithm: PseudonymizationAlgorithmsEnum
```

It could be converted to a string when we finally use it?